src/main/java/uk/gov/hmcts/divorce/caseworker/service/task/GenerateApplicant1NoticeOfProceeding.java

```diff
@@ -70,9 +72,10 @@ private void generateSoleNoticeOfProceedings(CaseData caseData, Long caseId) {
             log.info("Generating notice of proceedings for applicant solicitor for case id {} ", caseId);
 
             content = solicitorContent.apply(caseData, caseId, true);
-            templateId = caseData.getApplication().isCourtServiceMethod()
-                ? NFD_NOP_AS1_SOLEJOINT_APP1APP2_SOL_CS
-                : NFD_NOP_AS2_SOLE_APP1_SOL_SS;
+
+            var isCourtService = caseData.getApplication().isCourtServiceMethod();
+            templateId = isCourtService
```

Comment:
1 question - does solicitor service fall into the else statement? Is that okay?

Reply:
Resolved as this was discussed in Slack meeting and we believe this was already implemented and discussed